committee-code-of-conduct/charter.md

```diff
@@ -0,0 +1,148 @@
+the protection of our community and takes that duty seriously.
+
+## Communication with the committee
+The committee maintains a private mailing list for reporting incidents, asking
+confidential questions, and internal committee communication:
+
+[conduct@operatorframework.io][email]
+
+This email alias may not be the fastest path to a response in urgent situations.
+**If a response is time critical, reaching individual committee members via
+Slack is advisable. But an email must also be sent to
+[conduct@operatorframework.io][email] for tracking purposes**. This feedback helps
+guide the implementation of new policies and procedures.
+
+### Others acting on behalf of the committee
+Community moderation administrators in Slack, the mailing list, community
+events, and elsewhere are extensions of the committee and are able to be first
+responders to incidents. They are explicitly empowered to take those actions
+necessary to protect the community, especially when no committee members are
+available. All such events must be retroactively reviewed by the committee for
+appropriateness and consistency.
+
+Community chairs may also act in limited cases to enforce the code of
+conduct. Examples of this may be the deletion of GitHub comments, offensive
+Slack messages, or the eviction of bad actors from public meetings. These
+actions must be communicated to the committee for review.
+
+## Composition and Scope
+The [committee is composed of 5
+members](https://github.com/operator-framework/community/tree/master/committee-code-of-conduct).
+The committee is the primary recipient of all conduct complaints regardless of
+where in the community they originate. The only exception is at CNCF events,
+where the event Code of Conduct process supersedes this. That is primarily due
+to the high-impact nature of in-person violations and the need for more
+extensive staffing. This committee should be informed and consulted for all
+violations involving Operator Framework community members, regardless of circumstances.
+
+Additionally, the committee is responsible for drafting and executing on
+reporting, enforcement, and other policy matters. In most cases, policies are
+made public, however some materials will be confidential by nature of their
+content and application. As a general rule, the committee will provide as much
+transparency as possible, except in specific incident reports where no
+personally-identifying information about the reporter/reported will be shared.
+Anonymized aggregated incident data may be provided to the community as the
+committee sees fit.
+
+*The Operator Framework community project has explicitly delegated all Code of
+Conduct authority and enforcement to this committee. The committee can, at its
+discretion, delegate some authority to those tasked with enforcement.*
+
+## Election
+TODO: Our election processes will be outlined in a separate document, coming soon.
```

Comment:
isn't this provided on this PR with `election.md` ?

Reply:
Yes, needs to be discussed. Will remove this once it has been.